README.md

```diff
@@ -94,9 +119,63 @@ Example usage:
 
 ## Dtools
 
-Dtools are deprecated now, please use [godtools](https://github.com/PiwikPRO/godtools) instead.
+Dtools are deprecated now, please use [godtools](#godtools) instead.
+
+## Godtools
+
+[Godtools](https://github.com/PiwikPRO/godtools) is a tool for performing operations that require sharing sensitive credentials such as pulling docker images or s3 artifacts.
+
+### Login
+
+Allows to authenticate to docker registries.
+
+Example usage:
+
+```yaml
+---
+steps:
+  - name: Login to docker registries with godtools
+    uses: PiwikPRO/actions/godtools/login@master
+    with:
+      godtools-config: ${{ secrets.GODTOOLS_CONFIG }}
+      godtools-key: ${{ secrets.GODTOOLS_KEY }}
+      registries: acr, docker_hub
+```
+
+### Push
+
+Allows to push docker images to authenticated registries.
 
-Possible actions you can find [here](https://github.com/PiwikPRO/actions/tree/master/godtools).
+Example usage:
+
+```yaml
+---
+steps:
+  - name: Push container image
+    uses: PiwikPRO/actions/godtools/push@master
+    with:
+      image: piwikprocloud/astronauth
```

Comment:
```suggestion
      image: image/name
```

Reply:
I'd also reoder:
- setup
- login
- push

Worth to add that a login won't work without a setup so the usage (in fact) is:
```
steps:
  - name: Download and setup godtools
    uses: PiwikPRO/actions/godtools/setup@master
    with:
      godtools-config: ${{ secrets.GODTOOLS_CONFIG }}
      godtools-key: ${{ secrets.GODTOOLS_KEY }}
      reporeader-private-key: ${{ secrets.REPOREADER_PRIVATE_KEY }}
      reporeader-application-id: ${{ secrets.REPOREADER_APPLICATION_ID }}
  - name: Login to docker registries with godtools
    uses: PiwikPRO/actions/godtools/login@master
    with:
      godtools-config: ${{ secrets.GODTOOLS_CONFIG }}
      godtools-key: ${{ secrets.GODTOOLS_KEY }}
      registries: acr, docker_hub 
```